recipes/bitstruct/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "bitstruct" %}
+{% set version = "3.7.0" %}
+{% set sha256 = "708e4c043efea4e8ba8a02e203a3b02f270364b3580da44e89746de08fad08d7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - bitstruct
+
+about:
+  home: https://github.com/eerimoq/bitstruct
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Python bit pack/unpack package.'
+  description: |
+    simplejson is a simple, fast, complete, correct and extensible
```

Comment:
Copy-n-pasta :stuck_out_tongue_winking_eye:
Please add the description for bitstruct here.